### ISSUE TITLE
Abnormal circulating glycoprotein concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6008]
**Preferred term label:** Abnormal circulating glycoprotein concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** An abnormal concentration of glycoproteins in the blood.

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@JillMiller  there are so many glycoproteins with so many different functions -- I am not sure that it is good to have a general term. What is the use case?

**Comment by JillMiller** [comment]
I have a list of specific glycoprotein concentration abnormalities for possible new HPO terms. I suggested this to give them a parent term to keep them together within the abnormal protein level. Other than organization it would not be necessary!

**Comment by pnrobinson** [comment]
Probably it would be best to use just this as the super term
https://hpo.jax.org/app/browse/term/HP:0010876
Most circulating proteins have glycosylation, to date we have not differentiated.

**Comment by pnrobinson** [comment]
closing, I think this is not needed as a grouping term.
